app/models/license.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class License
+  def self.url(license_type)
+    case normalize_type(license_type)
+    when 'cc0 license'
+      'https://creativecommons.org/publicdomain/zero/1.0/'
+    end
+  end
```

Comment:
At some point we will need to add more licenses to this map but I could only find this one in our dataset so I only include one. Whatever values we want to account for we need to make sure that we match the ones used in DSpace.